### ISSUE TITLE
Get last event ID

### ISSUE THREAD

**Issue by cdvv7788** [getsentry/sentry-elixir#365]
https://docs.sentry.io/enriching-error-data/user-feedback/
How to make that work using elixir_sdk?

**Comment by mitchellhenke** [comment]
@cdvv7788 thanks for opening this! It's unfortunately not currently supported.

Doing it in Plug seems somewhat straightforward, but I think Phoenix will be a bit more difficult.

I will dig into this and see if I can come up with a solution.

**Comment by mitchellhenke** [comment]
@cdvv7788 I've put together a small prototype if you're able to and want to give it a try. You'll need to direct your Sentry dependency at the branch, and add the `collect_feedback: true` option to `Sentry.Plug`:

```elixir
# mix.exs
{:sentry, "~> 7.0", git: "https://github.com/getsentry/sentry-elixir.git", branch: "feedback"},

# my_app_web/router.ex
defmodule MyAppWeb.Router do
  use MyAppWeb, :router
  use Plug.ErrorHandler
  use Sentry.Plug, collect_feedback: true
```

When an error happens, it should show:
![image](https://user-images.githubusercontent.com/1430443/69666983-c95e8900-1052-11ea-9845-3b78d3bd1cfc.png)

**Comment by cdvv7788** [comment]
This works!. Any way to customize the form? I can live with the default one for a while, but I would like to be able to do that at some point.

Thanks for the help!

**Comment by mitchellhenke** [comment]
@cdvv7788 yeah, it's possible: https://docs.sentry.io/enriching-error-data/user-feedback/?platform=django#customizing-the-widget

I don't know on the best way to present that configuration through the Plug, especially if it's something more dynamic with user information, etc.

**Comment by cdvv7788** [comment]
Yes. I am not familiar with plug, so I can't be of help in there, sorry :(. For now this seems to work properly, and it is definitely a nice improvement.

**Comment by cdvv7788** [comment]
@mitchellhenke will this be merged to master? or further work is required?

**Comment by mitchellhenke** [comment]
@cdvv7788 I'd like to do some further work to become more familiar with the options and how to best incorporate them into the Sentry Plug module

If there are specific parts of the customization that are of more interest, I'd be happy to prioritize those

**Comment by cdvv7788** [comment]
For now it works for me. I just want to avoid pinning a branch (and later have my build fail when the branch dissapears) or pinning a commit and missing other updates to the plugin. If there is something I can help with, please let me know.

**Comment by mitchellhenke** [comment]
yeah, that's a good point. I think I will instead focus on publishing a version with minimal options to make this available in a stabler capacity, and defer further options until there's some more feedback in that area.

And thank you for the patience and continued communication on this 🙂

**Comment by cdvv7788** [comment]
@mitchellhenke I installed the latest version (7.2.2) and this feature seems absent. In github the last version is 7.2.1.
Is this still awaiting a release?

**Comment by mitchellhenke** [comment]
@cdvv7788 it was not included. I've added 7.2.2 to github and just released `7.2.3` that includes this feature

**Comment by cdvv7788** [comment]
Great. Thanks a lot.
